### PR TITLE
Quick fix for MessageID and Rollbar UUID

### PR DESCRIPTION
**Pull request by sophiemanagedbyq**

### CODE REVIEW

File: mbq/pubsub/consumer.py [hunk 1]
```diff
@@ -79,8 +79,9 @@ def process_queue(self):
                     )
             except Exception:
                 result = "failed"
-                logger.exception("An error occurred while processing the message.")
-                rollbar.report_exc_info()
+                logger.exception("An error occurred while processing the message. MessageId:" + str(body.get("MessageId", "default")) )
+                uuid = rollbar.report_exc_info()
```
Comment (sophiemanagedbyq) [inline mbq/pubsub/consumer.py]: let me know if this needs a try catch

File: mbq/pubsub/consumer.py [hunk 2]
```diff
@@ -79,8 +79,9 @@ def process_queue(self):
                     )
             except Exception:
                 result = "failed"
-                logger.exception("An error occurred while processing the message.")
-                rollbar.report_exc_info()
+                uuid = "1234564789"#rollbar.report_exc_info()
```
Comment (Pastromhaug) [inline mbq/pubsub/consumer.py]: remove this comment

File: mbq/pubsub/consumer.py [hunk 3]
```diff
@@ -79,8 +79,9 @@ def process_queue(self):
...
+                logger.exception("An error occurred while processing the message. MessageId:" + str(body.get("MessageId", "default"))
+                                 +  "\n <a href=\"https://rollbar.com/item/uuid/?uuid=" + uuid + "\">RollbarURL</a>")
```
Comment (Pastromhaug) [inline mbq/pubsub/consumer.py]: Use python f-string here

File: mbq/pubsub/settings.py [hunk 4]
```diff
@@ -161,3 +161,11 @@ def reload_api_settings(*args, **kwargs):
     setting = kwargs["setting"]
     if setting == "PUBSUB":
         project_settings.reload()
+
+import mbq.metrics
+mbq.metrics.init(
+    'pubsubtest',
+    env='testlocal',
+    constant_tags={'env': 'testlocal'}
+)
```
Comment (Pastromhaug) [inline mbq/pubsub/settings.py]: You'll need to run `black .` and `isort` to make your code formatted properly. All imports need to be at the top of the file to pass linting.

File: mbq/pubsub/consumer.py [hunk 5]
```diff
@@ -79,8 +79,13 @@ def process_queue(self):
...
             except Exception:
                 result = "failed"
-                logger.exception("An error occurred while processing the message.")
-                rollbar.report_exc_info()
+                uuid = rollbar.report_exc_info()
+                if uuid is None:
+                    uuid = "No UUID for RollbarFound"
+                messageId = body.get("MessageId", "default")
+                logger.exception(
+                    f'An error occurred while processing the message. MessageId: {messageId} \n <a href="https://rollbar.com/item/uuid/?uuid={uuid}">https://rollbar.com/item/uuid/?uuid={uuid}</a>'
```
Comment (mattbriancon) [inline mbq/pubsub/consumer.py]: No need to use HTML tags here. Papertrial doesn't HTML.

Also, newlines don't get escaped so the lines will get separated in Papertrail. Better to keep everything on one line so we know they're together.

File: mbq/pubsub/settings.py [hunk 6]
```diff
@@ -161,3 +163,6 @@ def reload_api_settings(*args, **kwargs):
     setting = kwargs["setting"]
     if setting == "PUBSUB":
         project_settings.reload()
+
+
+mbq.metrics.init("pubsubtest", env="testlocal", constant_tags={"env": "testlocal"})
```
Comment (sophiemanagedbyq) [inline mbq/pubsub/settings.py]: this was required to execute the testing docker container

**Pull request merged**